shared/test/unit/mocks/mock_mozContacts.js

```diff
@@ -18,7 +18,7 @@ MockMozContactsObj.prototype = {
           if (options.data.filterBy &&
             options.data.filterBy.indexOf('id') !== -1) {
             for (var i = 0; i < that.contacts.length; i++) {
-              if (options.data.filterValue === that.contacts[i].id) {
+              if (options.data.filterValue == that.contacts[i].id) {
```

Comment:
This not only makes me nervous, but also requires review from a Contacts peer. I'm assuming this is because we're passing a ```Number```-based ID somewhere, but code is expecting a ```String```-based ID, or *vice versa*. Have you determined if this is truly needed?